specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -0,0 +1,637 @@
+    "title": "Cognitive Service Form Recognizer API V1 (Preview)",
+    "description": "Extracts information from forms and images into structured data based on a model created by a set of representative training forms."
+  },
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/formrecognizer/v1.0-preview",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/custom/train": {
+      "post": {
+        "summary": "Train Model",
+        "description": "The train request must include a source parameter that is either an externally accessible Azure Storage blob container Uri (preferably a Shared Access Signature Uri) or valid path to a data folder in a locally mounted drive. When local paths are specified, they must follow the Linux/Unix path format and be an absolute path rooted to the input mount configuration\r\n setting value e.g., if '{Mounts:Input}' configuration setting value is '/input' then a valid source path would be '/input/contosodataset'. All data to be trained are expected to be under the source. Models are trained using documents that are of the following content type - 'application/pdf', 'image/jpeg' and 'image/png'.\"\r\n Other content is ignored when training a model.",
+        "operationId": "TrainCustomModel",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "trainRequest",
+            "in": "body",
+            "description": "Request object for training.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TrainRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/TrainResult"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Train new model example": {
+            "$ref": "./examples/Train.json"
+          }
+        }
+      }
+    },
+    "/custom/models/{id}/keys": {
+      "get": {
+        "summary": "Get Keys",
+        "description": "Use the API to retrieve the keys that were\r\n extracted by the specified model.",
+        "operationId": "GetExtractedKeys",
+        "consumes": [],
+        "produces": [
+          "application/json",
+          "text/json"
```

Comment:
Systemic issue; unless you actually intend to produce text/json, please don't include it (anywhere).

Reply:
Noted. Removed.